test_accounts/health_test_accounts.md

```diff
@@ -2,120 +2,124 @@
 
 **Veteran Health** test users have a variety of synthetic health records attached to them and can be used to test the [Veteran Health API](https://developer.va.gov/explore/health/docs/fhir?version=current). A small subset of these users can also be used for testing the [Clinical Health API](https://developer.va.gov/explore/health/docs/clinical_health?version=current).
 
-To log in to the sandbox environment with a test account, make sure you:
+**Do not modify any of the attributes for these test accounts, including passwords.** If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
 
-  1) Choose an account that is specific to the API you want to test.
-  2) Use ID.me to sign in to the sandbox environment.
-  3) **Don’t change any preselected answers** when asked about receiving an authentication code. Just click “Continue” to go to the next step.
+## Logging in with ID.me
 
-**Do not modify any of the attributes for these test accounts, including passwords.** If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
+To log in to the sandbox environment using ID.me, make sure you:
+
+  1. Choose an account that is specific to the API you want to test.
+  2. Select **ID.me** to sign in to the sandbox environment.
+  3. Enter the ID.me username and password (**Password1234!**).
+  4. **Don’t change any preselected answers** when asked about receiving an authentication code. Just click “Continue” to go to the next step.
+
+## Logging in with Login.gov
+
+To log in to the sandbox environment using Login.gov, make sure you:
 
-**Password for all ID.me accounts: Password1234!**
+  1. Choose an account that is specific to the API you want to test.
+  2. Select **Login.gov** to sign in to the sandbox environment.
+  3. Enter the Login.gov username and password (**Password12345!!!**).
+  4. Use the Login.gov MFA seed to generate a 2FA code using an application like Google Authenticator or Authy.
 
----
+## Veteran Account Information
 
```

Comment:
In between Veteran Account Information and the Notes, I would reiterate what the passwords are for ID.me and Login.gov accounts.
```suggestion
**Password for all ID.me accounts: Password1234!**
**Password for all Login.gov accounts: Password12345!!!**
```